src/project-2.js

```diff
@@ -47,61 +76,77 @@ const isPrime = (num) => {
 };
 
 const returnFirst = (arr) => {
+  return arr[0];
   // return the first item from the array
 };
 
-const returnLast = (arr) => {
+const returnLast = arr => arr.pop();
   // return the last item of the array
-};
 
-const getArrayLength = (arr) => {
+const getArrayLength = arr => arr.length;
   // return the length of the array
-};
 
-const incrementByOne = (arr) => {
+const incrementByOne = arr => arr.map(item => item + 1);
   // arr is an array of integers
   // increase each integer by one
   // return the array
-};
 
 const addItemToArray = (arr, item) => {
+  arr.push(item);
+  return arr;
+};
   // add the item to the end of the array
   // return the array
-};
+
 
 const addItemToFront = (arr, item) => {
+  arr.unshift(item);
+  return arr;
   // add the item to the front of the array
   // return the array
   // hint: use the array method .unshift
 };
 
 const wordsToSentence = (words) => {
+  return `${words.join(' ')}`;
```

Comment:
You could just do `return words.join(' ');`.  It automatically converts it into a string